### DISCUSSION TITLE
no me deja compilar

### DISCUSSION THREAD

**Discussion by diegoperez2004** [INGCOM-UNRN-P1/cursada-2023#88] [Preguntas y respuestas] [answered]
![image](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/141845473/774887dc-fe65-4a50-9b07-d4230cd2ca8b)
me salta este error cuando intento compilar

**Comment by diegoperez2004** [comment]
ya lo resolvi :D

**Reply by martinvilu** [reply]
¿Que era?

**Reply by diegoperez2004** [reply, marked as answer]
Estuve investigando y la causa pueden ser varias cosas pero una de esas es que no deja compilar porque hay algo ejecutandose y justo, antes de que me pasara esto, habia ejecutado un programa que quedaba imprimiendo un numero infinitamente y como no podia pararlo cerre el git bash y lo volvi a abrir, pero parece que aun asi el programa se estaba ejecutando en algun lado por lo que tuve que apagar y volver a encender la compu y ahi se arreglo.
